.vale/styles/AsciiDoc/ValidCodeBlocks.yml

```diff
@@ -0,0 +1,27 @@
+---
+extends: script
+ignorecase: true
+level: error
+message: "AsciiDoc file contains one or more unterminated code blocks."
```

Comment:
Since this only works if an attribute box is present featuring either "cols" or "options" specifiers, perhaps include this in the warning to help with troubleshooting. 
"AsciiDoc file contains one or more unterminated code blocks or you may be missing 'col' or 'options' table attributes." Could also link to AsciiDoc table specs? 
https://docs.asciidoctor.org/asciidoc/latest/tables/build-a-basic-table/